### PR TITLE
Fix some things.

### PR DESCRIPTION
**Pull request by Megabyte918**
- Fixed popsplits (https://github.com/Barbosik/MultiOgar/issues/333),
- Fixed cells flying through other cells (https://github.com/Barbosik/MultiOgar/issues/101)
- Fixed mothercells not able to eat viruses (https://github.com/Barbosik/MultiOgar/issues/322)
- Fixed mothercells not shooting fast enough (https://github.com/Barbosik/MultiOgar/issues/322)
- Reverted to old virus explosion-version with a few modifications. The current one is rarely seen in vanilla while this one is very similar to it. Fixes (https://github.com/Barbosik/MultiOgar/issues/373)
- Fixed randomized angles of small virus explosions (https://github.com/Barbosik/MultiOgar/issues/265)

No difference in performance.

### CODE REVIEW

**Comment by MastaCoder** [comment]
Barbosik is not alive anymore here :(

**Comment by NatsuTheGreat** [comment]
Lmfao. When a kid finds out how to do something XD

**Comment by GhostShot3** [comment]
@NatsuTheGreat  Who ?

File: src/GameServer.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ function GameServer() {
         playerMinSplitSize: 60,     // Minimum player cell size allowed to split (mass = 60*60/100 = 36) 
         playerStartSize: 64,        // Start size of the player cell (mass = 64*64/100 = 41)
         playerMaxCells: 16,         // Max cells the player is allowed to have
-        playerSpeed: 1,             // Player speed multiplier
```
Comment (MastaCoder) [inline src/GameServer.js]: NO NO NO. Do not make the cells slower, 1 is vanilla enough.

Comment (Megabyte918) [inline src/GameServer.js]: The increase i did with the normalized distance in PlayerCell.js doubles the player speed, so reducing the playerspeed config by half in gameserver.ini balances it out to what it originally was. Popsplits wont work well without it.

**Comment by Barbosik** [comment]
incorrect

**Comment by Megabyte918** [comment]
Can you please explain?

**Comment by NatsuTheGreat** [comment]
BARBOSIK YOU'RE ALIVE! Have you got any ideas on how to fix the skin issue?

**Pull request closed without merging**